README.md

```diff
@@ -1,23 +1,74 @@
+
+## Summary
+
 
 Abide by Wheaton’s Law: don’t be a dick.
 
 Everyone is welcome to participate, and no judgement will be made on members of the greater Mac community who are not a part of the community.
 
+## The Details:
+
+
+### Policies
+
+The MacAdmins Slack Team is dedicated to providing a harassment-free experience for everyone, regardless of (but not limited to): gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or preferred technological ecosystem. 
+
 Be welcoming of everyone: discrimination based on gender identity, race, religion, nationality, political beliefs, sexual preference, preferred technological ecosystem, and concerns that have not been included in this list will be addressed with progressive discipline based on the severity of the infraction as determined by the administrative team.
 
-Harassment, hate speech, bullying, sea-lioning, and gas-lighting are prohibited activities and are grounds for suspension from the community when there is clear documentation in public channels. If you experience any of these behaviors in direct messages please immediately contact the administrative team.
+Harassment, hate speech,  and bullying are prohibited activities and are grounds for suspension from the community when there is clear documentation in public channels. If you experience any of these behaviors in direct messages please immediately contact the administrative team.
 
-Your direct messages are only viewable by yourself and the other involved party and no one else. That being said, this is a community for professional communication and should be treated accordingly, regardless of the number of animated GIFs.
+Harassment includes, but is not limited to:
 
-If you have concerns regarding this Slack instance please feel free to contact the administrative team and we will work to understand and address them respectfully and with an appropriate level of privacy to the given situation.
+* Offensive and/or unwelcome comments and behaviors related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or preferred technological ecosystem.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following 
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Physical contact and simulated physical contact (eg, textual descriptions like “hug” or “backrub”) without consent or after a request to stop
+* Unwelcome sexual attention including gratuitous or off-topic sexual images or behaviour
+* Hate speech
+* Repeatedly asking questions of an individual or targeted group, even if remaining calm/civil while doing so (aka ["Sea-lioning"] (http://nancyfriedman.typepad.com/away_with_words/2014/11/word-of-the-week-sea-lioning.html))
```

Comment:
It'd probably be good to add in dog piling as well (https://en.wiktionary.org/wiki/dogpile) as it is the non-interrogative variation of sealioning. I realize it didn't get added into the initial draft of the CoC, and really should have been included.